plugins/grpc/go/v1.3.0/Dockerfile

```diff
@@ -69,11 +73,10 @@ index 340eaf3e..7f0fba56 100644
 EOF
 WORKDIR /tmp/grpc-go/cmd/protoc-gen-go-grpc
 RUN --mount=type=cache,target=/go/pkg/mod \
-    CGO_ENABLED=0 \
-    go build -ldflags "-s -w" -trimpath
+    go build -o protoc-gen-go-grpc -ldflags "-s -w" -trimpath
```

Comment:
Should we update this to be like the other go plugins, i.e., `&& mv /go/bin/${GOOS}_${GOARCH}`

Reply:
This one is different because we're patching it to support outputting to a separate package.

Reply:
Hehe, as soon as I hit reply, I remembered about the patch 😅